src/main/scala/amba/apb/Protocol.scala

```diff
@@ -9,8 +9,8 @@ object APBParameters
   // These are all fixed by the AHB standard:
   val protBits  = 3
 
-  def PROT_PRIVILEDGED = UInt(1, width = protBits)
+  def PROT_PRIVILEGED  = UInt(1, width = protBits)
   def PROT_NONSECURE   = UInt(2, width = protBits)
   def PROT_INSTRUCTION = UInt(4, width = protBits)
-  def PROT_DEFAULT = PROT_PRIVILEDGED
+  def PROT_DEFAULT = PROT_PRIVILEGED
```

Comment:
Thanks for this typo fix:
However for the happiness of RocketChip downstream, would you mind adding a function alias to not break the rocket-chip downstream?:
```
@deprecated(xxx)
def PROT_PRIVILEDGED = PROT_PRIVILEGED
```

Reply:
I remember in the last dev meeting, we decided not maintain API stability so aggressively.